nautilus/yaml_files/kube_dev_suite_lris_slitmask.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: batch/v1
```

Comment:
I don't think we want these yaml files in git, since they become obsolete once their branch is deleted.

Reply:
gone